tests/e2e/homework/pages/SelectFlight.page.ts

```diff
@@ -0,0 +1,150 @@
+import { Page, expect, Locator, test } from '@playwright/test';
+import { vars } from '../others/constants';
+import { BasePage } from './BasePage.page';
+import { LoginPage } from './Login.page';
+
+export class SelectFlightPage extends BasePage {
+    readonly loginPage: LoginPage;
+
+    // Buttons and Inputs locators
+    readonly tripTypeButtonReturn: Locator;
+    readonly tripTypeButtonOneWay: Locator;
+
+    readonly selectFrom: Locator;
+    readonly selectTo: Locator;
+
+    readonly selectDepartDay: Locator;
+    readonly selectDepartMonth: Locator;
+
+    readonly selectReturnDay: Locator;
+    readonly selectReturnMonth: Locator;
+
+    readonly continueButton: Locator;
+    readonly selectFlightButton: Locator;
+
+    readonly firstFromCity: Locator;
+    readonly firstToCity: Locator;
+    readonly secondFromCity: Locator;
+    readonly secondToCity: Locator;
+
+    constructor(page: Page){
+        super(page);
+        this.loginPage = new LoginPage(page);
+        
+        this.tripTypeButtonReturn = page.locator('[value="return"]');
+        this.tripTypeButtonOneWay = page.locator('[value="oneway"]');
+        this.selectFrom = page.locator('[name="fromPort"]');
+        this.selectTo = page.locator('[name="toPort"]');
+        this.selectDepartDay = page.locator('#departDay');
+        this.selectDepartMonth = page.locator('#departMonth');
+        this.selectReturnDay = page.locator('#returnDay');
+        this.selectReturnMonth = page.locator('#returnMonth');
+        this.continueButton = page.locator('[type="submit"]');
+        this.selectFlightButton = page.locator('[type="checkbox"]');
+
+        this.firstFromCity = page.locator('#container > div:nth-child(4) > b:nth-child(2)');
```

Comment:
These locators are considered bad. A few changes to frontend (e.g. not bold but underlined text) and the test is broken, even though functionality of it did not change.
For the places you want to test - in best case scenario we'd ask devs to add test id on those elements.

Other example - we could take the full div element, it's text, and removing extra chars if any, and then asserting full string of text.